Assets/Scripts/Game/Utility/PopulationManager.cs

```diff
@@ -279,6 +288,7 @@ int CreateNewPoolItem()
             PoolItem poolItem = new PoolItem();
             poolItem.npc = npc;
             poolItem.npc.Motor = motor;
+            poolItem.npc.Asset = motor.MobileAsset;
```

Comment:
Good idea. This is a better place to set `Asset` rather than use `GetComponentInChildren`. :)